src/Card/Card.tsx

```diff
@@ -13,21 +15,26 @@ import CardThumbnail, { CardThumbnailProps } from './CardThumbnail';
 const DEFAULT_CARD_HEIGHT = 130;
 
 const StyledCard = styled(MuiCard, {
-  shouldForwardProp: (prop) => prop !== 'elevation' && prop !== 'fullWidth',
-})<{ isOver: boolean; fullWidth?: boolean; elevation?: boolean }>(
-  ({ theme, elevation, fullWidth, isOver }) => ({
-    borderRadius: theme.spacing(1),
-    boxShadow: elevation ? theme.shadows[2] : '0px 2px 2px #eeeeee',
-    width: fullWidth ? '100%' : 'max-content',
-    maxWidth: '100%',
-    border: isOver ? '2px solid black' : 'none',
-  }),
-);
+  shouldForwardProp: (prop: string) =>
+    !['elevation', 'fullWidth', 'isSelected', 'isOver'].includes(prop),
```

Comment:
Should we extract the array in a const ?

```suggestion
    !PROPS_TO_FORWARD.includes(prop),
```